.tool-versions

```diff
@@ -1,3 +1,3 @@
-nodejs 18.11.0
+nodejs 18.10.0
```

Comment:
We should wait node-build merging for latest 🤔 

https://github.com/nodenv/node-build/pull/756